components/org.wso2.carbon.identity.application.authenticator.biometric/src/main/resources/Artifacts/wait.jsp

```diff
@@ -0,0 +1,135 @@
+<%--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  ~
+  --%>
+
+<%String commonauthURL = "https://biometricauthenticator.private.wso2.com:9443/commonauth";%>
+
+<%@ taglib prefix = "s" uri = "http://java.sun.com/jsp/jstl/core" %>
+<%@ page language = "java" contentType = "text/html; charset=UTF-8" pageEncoding = "UTF-8" %>
+<%@ page import="org.wso2.carbon.identity.application.authenticator.biometric.BiometricAuthenticatorConstants" %>
+
+<html>
+<head>
+    <meta http-equiv = "X-UA-Compatible" content = "IE=edge">
+<%--    <meta charset = "utf-8">--%>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WSO2 Identity Server</title>
+    <meta charset="UTF-8">
+    <meta charset="utf-8"/>
+
+    <link rel="icon" href="images/favicon.png" type="image/x-icon"/>
+    <link href="libs/bootstrap_3.3.5/css/bootstrap.min.css" rel="stylesheet">
+    <link href="css/Roboto.css" rel="stylesheet">
+    <link href="css/custom-common.css" rel="stylesheet">
+    <link href="css/loading1.css" rel="stylesheet">
+
+    <script language="JavaScript" type="text/javascript" src="libs/jquery_1.11.3/jquery-1.11.3.js"></script>
+    <script language="JavaScript" type="text/javascript" src="libs/bootstrap_3.3.5/js/bootstrap.min.js"></script>
+
+    <header class="header header-default">
+        <div class="container-fluid"><br></div>
+        <div class="container-fluid">
+            <div class="pull-left brand float-remove-xs text-center-xs">
+                <a href="#">
+                    <img src="images/logo-inverse.svg" alt="WSO2" title="WSO2" class="logo">
+                    <h1><em>Identity Server</em></h1>
+                </a>
+            </div>
+        </div>
+    </header>
+</head>
+<body>
+<h2>Please check your mobile device and authenticate with the fingerprint</h2>
+<div class="loader"></div>
+<form id="toCommonAuth" action="<%=commonauthURL%>" method="POST" style="display:none;">
+    <label for="sessionDataKey">sessionDataKey</label><input id="sessionDataKey" name="sessionDataKey">
+    <label for="signedChallenge">signedChallenge</label><input id="signedChallenge" name="signedChallenge">
+</form>
+</body>
+
+<%--// TODO: make a war file without putting to auth endpoint-Discuss with Pulasthi ayya.
+ --%>
```

Comment:
What is the status of this TODO?

Reply:
Told to put the wait.jsp file to IS authentication endpoint.